Components/buttons/button.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/installGroot.html/install.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Buttons</title>
+</head>
+<body>
+    <div class="groot">
+        <div class="groot-container first">
+            <nav>
+            <img src="/images/groot.png" style="width: 4rem;height: auto; margin: 0 0.5rem "/>
+            <span style="display: inline-flex;"><a href="/" class="square">GrooT UI</a></span>
+            <div class="version"> v1.0</div>
+            </nav>
+            <div class="item-container">
+                <div class="install-ui"><a href="/installGroot.html/install.html" class="install-text"> Installation </a></div>
+                        <a href="/Components/avatar/avatar.html" class="avatar-text ">Avatars</a>
+                        <a href="/Components/alert/alert.html" class="avatar-text">Alert</a>
+                        <a href="/Components/badges/badges.html" class="avatar-text">Badges</a>
+                        <a href="/Components/buttons/button.html" class="avatar-text">Buttons</a>
+                        <a href="/Components/card/card.html" class="avatar-text">Cards</a>
+                        <a href="/Components/modal/modal.html" class="avatar-text">Modals</a>
+                        <a href="/Components/input/" class="avatar-text">Input</a>
+                </div>
+                </div>
+        <div class="groot-container second">
+            <div class="avatar-container">
+                <span style="font-size: 2.5rem;">Button</span><br/>
+                <span style=><p style="font-size: 1.3rem; color: grey;border-left: 3px solid #6B7280;margin-left: 1rem;">The Button component is used to trigger an action or event, such as submitting a form, opening a dialog, canceling an action, or performing a delete operation.</p></span>
+            </div>
+            
+            <span style="font-size: 2rem; margin-left: 3rem; color: grey;">Button Primary</span><br/>
+            <div class="avatar-box">
+            <div class="badges">
+                <button class="button-first button-first-color"> ADD TO BAG </button>
+                <button class="button-third button-third-color"> CONFIRM </button>
```

Comment:
You might want to consider reducing the whitespace between the title and the component below to make it look more categorized.